Updating the metrics doc based on customer feedback (#2161)
fishtown-analytics/docs.getdbt.com: website/docs/docs/building-a-dbt-project/metrics.md

| grain       | `day`, `week`, `month`, `quarter`, `year` | The time grain that the metric will be aggregated to in the returned dataset | Required |
| dimensions  | [`plan`, `country`] | The dimensions you want the metric to be aggregated by in the returned dataset | Optional |
| secondary_calculations  | [`metrics.period_over_period(comparison_strategy="ratio", interval=1, alias="pop_1wk")`] | Performs the specified secondary calculation on the metric results. Examples include period over period calculations, rolling calcultions, and period to date calculations. | Optional |
| start_date  | `2022-01-01` | Limits the date range of data used in the metric calculation by not querying data before this date | Optional |
| end_date    | `2022-12-31` | Limits the date range of data used in the metric claculation by not querying data after this date | Optional |
| where       | `plan='paying_customer'` | A sql statment, or series of sql statements, that alter the **final** CTE in the generated sql. Most often used to limit the data to specific values of dimensions provided | Optional |

#### Secondary Calculations

> Secondary calculations are window functions which can be added to the metric calculation and performed on the primary metric or metrics. You can use them to compare values to an earlier period, calculate year-to-date sums, and return rolling averages. Custom secondary calculations can be added into dbt projects - for more information on this, reference the [package README](https://github.com/dbt-labs/dbt_metrics#secondary-calculations).

Reviewer: switch to active voice and what do you mean by 'performed' @callum-mcdata ?

```suggestion
Secondary calculations are window functions you can add to the metric calculation and perform on the primary metric or metrics. 

You can use them to compare values to an earlier period, calculate year-to-date sums, and return rolling averages. You can add custom secondary calculations into dbt projects - for more information on this, reference the [package README](https://github.com/dbt-labs/dbt_metrics#secondary-calculations).
```
